biz.aQute.tester/src/aQute/tester/plugin/ProjectTesterImpl.java

```diff
@@ -83,6 +83,14 @@ protected void updateFromProject() throws Exception {
 
 		launcher.addRunBundle(me.getFile()
 			.getAbsolutePath());
+
+		//
+		// We also add it to the project to handle the case of the remote
+		// launcher that wants to sync the bundles with the runtime dynamically
+		//
+
+		getProject().getRunbundles()
```

Comment:
This seems an unwanted side effect. We already add `me` to the launcher above. Jamming into the project seems wrong. Every time you launch, you will add it again, no? I think some other solution is needed.

Reply:
There's no other way. the remote launcher is blowing away the runbundles set using `launcher.addRunBundle`.
 I'm open to other options.

Reply:
- the tester gets the launcher instance
- the tester calls prepare() where it adds the tester bundle to launcher runbundles
- the tester then calls getProjectLauncher().launch(); which calls launcher.prepare() which calls launcher.updateFromProject(); which calls runbundles.clear();

Also, RemoteProjectLauncherPlugin inherits from aQute.bnd.build.ProjectLauncher which has https://github.com/bndtools/bnd/blob/master/biz.aQute.bndlib/src/aQute/bnd/build/ProjectLauncher.java#L148

Reply:
fixed this